config/serverless.security.search_ai_lake.yml

```diff
@@ -7,8 +7,9 @@ xpack.features.overrides:
   ### The following features are Security features hidden in Role management UI for this specific tier.
   securitySolutionTimeline.hidden: true
   securitySolutionNotes.hidden: true
+  siemV2.hidden: true
```

Comment:
Why do we need to hide this?

Reply:
It is so that we hide `security` sub privileges in the roles UI for ai4soc. Correct?

Reply:
This hides the entire _Security_ feature. Is that intentional?

Reply:
No it doesn't hide the entire Security feature (as you see in the screenshot) it hides only the security sub-feature that has privileges such as host isolation, response actions, trusted apps etc.

Here's the [parent issue](https://github.com/elastic/security-team/issues/11781) and a [breakdown issue ](https://github.com/elastic/security-team/issues/12040)

Reply:
I see, you want to hide the features:
<img width="754" alt="Captura de pantalla 2025-04-09 a les 10 29 02" src="https://github.com/user-attachments/assets/be3bdaf5-524f-4186-bf09-6a4c4a36386c" />
<img width="754" alt="Captura de pantalla 2025-04-09 a les 10 30 47" src="https://github.com/user-attachments/assets/14ff101b-6939-43f3-90ce-0cf1b9858a4b" />

Reply:
https://github.com/elastic/security-team/issues/11781#issuecomment-2788825208

Reply:
Oh I see, it's a "feature" like you call it and not sub-feature. 

I actually see it as Security "feature" (not category)  with a list of sub-features within it, one of which is a Security "sub-feature" with list of "privileges". Same with Management feature and a list of privileges listed under it.

But either way the idea is to hide the Security "feature" (sub-feature as I call it).

Reply:
Oh okay I see what you mean with the alerts not working as the Security feature controls other areas of security as well. I'll then go ahead and try and hide the "sub-feature" privilege (same as in your screenshots)